addons/spreadsheet_dashboard/static/tests/dashboard/dashboard_action.test.js

```diff
@@ -40,6 +40,27 @@ function getServerData(spreadsheetData) {
     return serverData;
 }
 
+async function openCogMenu() {
+    const cogIcon = getFixture().querySelector(
+        ".o_search_panel_category_value .o_cog_menu_container .fa-cog"
+    );
+    await click(cogIcon);
+    await animationFrame();
+    return cogIcon;
+}
+
+async function clickEditOption() {
+    const popover = getFixture().querySelector(".o_popover");
+    const editOption = popover.querySelector(".o-dropdown-item:first-child");
```

Comment:
```suggestion
    const editOption = getFixture().querySelector(".o_popover .o-dropdown-item:first-child");
```
can't you merge the selectors ? Also this helper and the one for delete seems a bit overkill, they are very simple and only use once. I'd inline the code